### ISSUE TITLE
Brand Message

### ISSUE THREAD

**Issue by adj-gh** [turtlecoin/meta#56]
Hello guys! Today I come to you with an issue I think is important to all the community.
We need to start thinking about what approach the project is gonna have in terms of what way we're gonna spread TurtleCoin's message. 

I have asked around a few times and it seems like the community has a lot of newcomers to the whole crypto world, myself included.
I would feel like the _Educational cryptocurrency_ is one of the best approaches at gettings TurtleCoin into peoples minds, as a lot of material around the web tends to be overly complicated and confusing. 

A lot of work from all the current teams would be needed, as all projects should mantain and strive to achieve that simple and friendly approach.

Mision and vision are cliche, but a cliche that works. So putting one together would be of utmost importance to achieve this unification.
Bring your opinions and suggestions please!

**Comment by timmcleod** [comment]
Good stuff. Maybe cliche, but definitely important. Also, "values"...what are the values, or the guiding principles, of the leadership of this effort? After mission + vision + values are defined, they can start to inform some of the other aspects of the brand... i.e. brand personality and voice, functional and emotional benefits of TRTL vs other, and even the brand's promise to it's users. 

I'm not a branding expert, but it's fresh on my mind after (mostly) watching some talented folks from the sidelines execute a branding effort with my company recently.

**Comment by BebopTC** [comment]
I believe vision -> mission -> objectives/goals is the hierarchy. Values apply and bleed throughout. Vision is the big picture, how we see the ideal future. Mission is how we want to realize the vision. Objectives/goals are the steps carried out to achieve the mission. Values are the guiding principles that shape the way we define our goals and objectives and the way in which we carry them out... the modus operandi so to speak.
As a rough draft brain dump, I'd say our vision is a worldwide ecosystem of digital and physical exchange and commerce facilitated by turtlecoin. Mission is to build the tools and services that are the foundation of the ecosystem and to make contribution accessible to everyone interested. Values are fun, memes, openness, transparency, inclusiveness, pizza, initiative, cooperation, generosity, privacy, security, technical prowess.
I'll leave out objectives/goals for the moment

**Comment by swampfighter** [comment]
There's a pretty standard marketing 'system' that is really useful here, because it's going to get everyone on the same page.  I mention this because I work for a marketing agency, and we use this approach for every single one of our clients.  

Goal - The big picture. Utopia, whatever you want to call it. 
Our goal is to grow Turtlecoin into a widely-accepted, recognized cryptocurrency. 

Objective - Something we do to reach the goal. Something measurable. This brings people down to earth. 
Example: Grow the userbase of turtlecoin by 50% by July 2018.  (I'm just picking a number. How do we measure userbase now?)

Strategy - Specifc ways to grow userbase. We can, and should have multiple strategies towards meeting that objective of usergrowth. Below is just one example strategy.
Example: Increase visibility on reddit. 

Tactics - the specific efforts we will take to effect the strategy. (Increase visibilty on reddit.) 
1. Create reddit turtle tipbot and encourage use of bot both inside and outside of cryptocurrency subreddits.  Optimal targets may include ELI5, TIL, and other high-visibility subreddits. 
2. Create a small team of astrotufers to make offhand remarks about turtlecoin in relevant cryptocurrency subreddits. 
3. Repurposing/creating memes from outside the popculture norm (and encouraging their use). The 'norm' seems to be Teenage Mutant Ninja Turtles, that's great, but there are others like "Turtles, all the way down" 
4. ?

**Comment by timmcleod** [comment]
@swampfighter Marketing systems or strategies like you've described are really about delivering the message of the brand. Before getting that specific, the brand message itself needs be clearly defined. Maybe that's already done? I've seen the brand guides, but I thought this thread was about clarifying the brand message with mission/vision statements.

By the way, I just sort of jumped in here. Sorry about that. I've been poking around and love what you guys are doing with TurtleCoin. 👍

**Comment by swampfighter** [comment]
I guess you can approach it in two ways, but I still feel that both ways follow the same format. 

Approach 1:
Goal: Define the turtlecoin brand. 
Objective: Create a unified message consisting of a vision/whitepaper and mission statement along with other possible supporting materials. 
Strategy: Corner RockSteady and Bebop and force them to reveal their cryptocurrency ideology.
Tactics: Offer icecream with sprinkles. 

Goal: grow Turtlecoin into a widely-accepted, recognized cryptocurrency.
Objective: Grow Userbase by 50%
Strategy: Create a unified message consisting of a vision/whitepaper and mission statement along with other possible supporting materials that inspire and attract users.
Tactics: focus groups, A/B testing, etc.

**Comment by timmcleod** [comment]
Sounds like you’re leaning towards approach 2. Branding and marketing aren’t 2 different approaches... just stages of the process. Just offering my 2 cents. I’ll let ya’ll work.

**Comment by swampfighter** [comment]
I'm not trying to be flip,  just giving examples of how GOST works. It's not just limited to marketing, you can use it for managing any goal/task.  

Organized project management is something near and dear to my heart. Turtlecoin is an interesting idea that I'd like to see succeed.

**Comment by BebopTC** [comment]
the big picture is the vision.  
the mission realizes the vision.  
the goals/objectives are the things carried out to accomplish the mission.  
values are the principles we adhere to.  
those are the things we communicate outward.

Seems to be confusion.  
I believe what @swampfighter is referring to is a technique/system/process for sensibly [internally] defining, communicating & completing goals & objectives. It is not what we communicate outward, as a brand. nothing wrong with it but i don't see how we need that workflow or degree of formality here in defining how we communicate the brand identity.

**Comment by BebopTC** [comment]
Also @swampfighter duuude I didn't mean to imply there isn't value in the technique you suggested I'm just not sure how it would apply to the discussion here, but I may be missing something. I hope I didn't seem harsh or unwelcoming! Were you suggesting it as a way for managing our marketing objectives/goals, or even goals in general?

That makes more sense to me but if that is the case, I would say it's a bit outside of this thread. However I think you will find the way we leverage github, particularly in how we use the Issues feature for task management, provides us a very fluid way of defining tasks, discussing them, and managing them.  I don't want to derail the discussion of brand identity into one about project management as I have some other stuff to say about identity, but I will say we really strive to be a very accessible, decentralized and open project. To that end you are free to open issues that correspond to the higher level resolution, i.e. goals, and you can write the issue in the GOST format if you'd like. From there have discussion with whoever in the community is interested and come up with or review/tweak the objectives and strategy and tasks. Then you can make new issues to represent the individual tasks you've come up with. 

You don't have to follow that pattern,  but I think it makes pretty good sense with how we operate. Ultimately mapping work to issues works well with our typical use patterns and the tooling of github supports an issue-based workflow. You can cross reference Issues by simply typing their number (e.g. #56) to keep things tied together, anyone can assign themselves to indicate they are working on it, and once it is completed can be reviewed and archived. We prefer to keep deliverables in git repos, even if they are not actually "programs", so that they are accessible, and so that there is a sort-of permanent history of change (commits). Commits can be referenced in issues, and vice versa. 

If you want to talk more about that, open another issue about it and we'll carry on there.

**Comment by finnious** [comment]
Looping back to the original @adj-gh issue...

> what way we're gonna spread TurtleCoin's message.

> best approaches at getting TurtleCoin into peoples minds

> Mission and vision are cliche, but a cliche that works.

I have action items in mind for user growth. They are directed at two audiences. 

1. The "Crypto aware" - The people with an awareness of cryptocurrencies and will compare and contrast Turtle to other coins. Day Traders, buy and holders, developers, you get the picture.
2.  The TurtleCoin is the gateway to Crypto - The people that will go directly to the educational material and will most likely get overwhelmed in a hurry.

The two crowds exist in different places online, speak differently online and want/need different content.
We need to establish the Vision & Mission before heading in two directions to grow these two audiences.

As @BebopTC metioned:
> the big picture is the vision.
> the mission realizes the vision.

> As a rough draft brain dump, I'd say our vision is a worldwide ecosystem of digital and physical exchange and commerce facilitated by turtlecoin. Mission is to build the tools and services that are the foundation of the ecosystem and to make contribution accessible to everyone interested. Values are fun, memes, openness, transparency, inclusiveness, pizza, initiative, cooperation, generosity, privacy, security, technical prowess.

That rough draft is targeted more towards the "Cryto Aware" and less inclusive of the Crypto Noob.

My question to the Turtle Marketing Crowd(@timmcleod @swampfighter): Do we make the Vision/Mission more inclusive or worry about the audience wording on the objectives/goals level?

My opinion is a more inclusive Vision/Mission.

**Comment by BebopTC** [comment]
Good point @finnious about how we target different audiences ,and I agree those are our two broad dems. While we may adjust to some degree how and what specifically we communicate to each, generally speaking, it's very important that we never compromise our vision and our values - they must HODL. Some aspects may be more emphasized in one setting than another, just like we adjust ourselves between the office and the dinner table, but our identity should be consistent.

**Comment by bd-ongithub** [comment]
Mission and Vision are good starting points, and I think some good insights have been posted so far. 

I'm a bit late joining into the thread but I figured it wouldn't hurt touching upon some branding basics, and that comes around to Ye Olde Branding Questionnaire.

A little background on this first: a lot of these questions are very often used when generating a creative brief, which is basically a jumping off point for branding projects, general commercial creative work, and full blown campaigns. I think a lot of these have already been answered more than once, but it might be helpful to keep them all linked together in a single document everyone can reference (with special attention looking forwards, some of these questions are aimed at informing someone who is completely new/doesn't know about the project).

Additionally, some of these questions may appear to have a very obvious answer at first, but the key here is to dig as deep as possible into the most basic, simplified answers. This process is usually very helpful moving forward, since it can help unearth some unique insights in what we communicate, and how we can communicate it in the most effective manner possible.

To sum it up, a finalized creative brief is meant to inspire as much as it is aimed at informing creatives involved (or wanting to get involved) in the project to plot the best way to communicate a brand to a specific audience in ways that are relatable as well as desirable to encourage adoption.

Here goes:
1) What does TurtleCoin do? 
_You have 30 seconds to explain the project to potential investor/adopter, what would you say?_

2) What unique problem do you solve? How do you solve it?
_What is the purpose of TurtleCoin's existence (other than to make money)?_

3) Who are your 3 main competitors (direct or indirect)? What do you like about their presence? What do you dislike about their brand identity?
_We don't exist in a void, of course. It also doesn't mean our focus is to combat other brands, but it is good to keep projects in mind that can be learned from as TurtleCoin does its own thing._

4) What is TurtleCoin's Unique Selling Proposition?
_I think Mission and Vision fits in here very nicely. To learn more about a USP this blog goes over it nicely with some big brand examples: https://blog.kissmetrics.com/unique-selling-proposition/_

5) Is there a unique story behind your business or business name or logo? 
_I'll go ahead and throw in a big resounding YES! here ;) now just how much detail we can add?
People connect through stories and storytelling is an effective way to create and sell a brand image. The story creates the customer.
A powerful story can also be a great execution for a creative piece down the road. As an example, check out Johnnie Walker's ad: The Man Who Walked Around the World https://www.youtube.com/watch?v=fZ6aiVg2qVk_

6) Share 5 adjective or words that best describe your company
_Since we have fast, easy, safe, and fun, maybe make it 5 additional ones. Think of TurtleCoin as a person/creature/entity, what adjectives would you use to describe him/her/it to a friend?_

7) Describe your ideal TurtleCoin adopter/user.
_In this case, who is our ideal TurtleCoin user? I know it's tempting to say everyone, but getting specific here can help a lot. Really try to imagine an exact person if you can (again, think of it as if you're describing this person to a close friend): Is it a man/woman? How is he/she? What profession(s) are they involved in? How do they dress? What kind of stuff do they like to do in their spare time? What kind of things do they consider important in their lives?_

8) What is the primary message do you want to convey to an adopter/user? 
_Keep in mind this doesn't have to be verbal at all. This is about how we want customers to feel each time they interact with TurtleCoin. ex: How do people feel each time someone mentions an iPhone?_

9) Share 5 adjective or words that best describe your ideal adopter/user.
_Think adopter/user description lite. Best friend scenario again, you're about to introduce them to your ideal user, what kind of brief prep do you give your friend to get them ready to meet this person?_

There's more, but I'd like to keep it up to here for now to see what you guys think. If you guys find it a bit too boring or not too useful, then I just gotta give you guys thanks for checking this out. Hopefully it helps bring forth some extra insights :)

**Comment by BebopTC** [comment]
let's talk about identity. we want our identity to come through not only in what our vision and mission are, but in how we describe and represent them. but first we have to understand what our identity is.

think about our discord. think about all our silly antics.. the ninja turtle mythos, the memes, the birth on /biz/, the raindances and the rain parties, the laughs and the jokes.

think about the event fliers, with the trippy bright colors and goofy imagery and comical fonts, reminiscent of party flyers from 90s raves, that make you feel excited and invited and wondering what obnoxios fun is in store.

think about exile. turtle marine. the footclan. the guerillas. our emojis. we look like a bunch of goons and mishaps, and that's exactly what we want to be.

remember the foot clan hideout in the 1990s ninja turtles movie? what, you haven't seen it? stop what you're doing right now and go watch. required reading. http://zmovies.to/teenage-mutant-ninja-turtles-1990-full-movie/

we're the rag tag overlooked yeterday's kids of tomorrow. on the surface we may look like a crowd of fuckary and chaos, but when it comes down to business we're serious motherfuckers. we're deep-thinking and masterful about the things that matter most to us: our crypto, our code, our privacy, our network, our memes, our freedom. we're cryptographers and programmers, engineers and economists, system admins and mathemeticians, designers and artists, builders of the future economy. we're that purple-mohawk, septum-pierced pot smoking hacker who runs all the security at Megacorp 'cause no one else is as good. we're the 8 bit generation, east meets west, digital age cold war secret agents of a borderless nation. we're cereal killer and phantom phreak. we're the girl with the dragon tattoo.

we're like nintendo game boy meets nickelodean ooze meets adult swim meets linus torvalds meets 4 chan meets half life. we are products of the informaton age whose dreams and religion were shaped by saturday cartoons and super metroid and sonic and outrun, whose bedtime reading was the Manifesto (http://phrack.org/issues/7/3.html) and 2600. our heros are the likes of rms, esr, alan turing, dpr, leonard adelman, the pizza dude from that place on 3rd street with the big ass pepperonis, aeon flux, the wu tang clan, cdc ninja strike force and satoshi nakamoto.

we are not dry, overly redacted and edited, depersonalized stuffy suit-and-tie cogs in a corporate machine. we are not 'play-it-safe' fast-forward marketing designed to please everyone by adhering to the lowest common denominator. we are not neat and tidy and consistent and manufactured. we do not bend and shift our personalities or our principles to conform under the pressures of the world; rather, we bend and shift the world to conform to the pressure of our personalities and principles.

we are not a meme coin, we are THE meme coin. we are the wolves of doge street. our roots run deep and our reach far. we're here to break every convention and think outside of the box. we are a network of shelled troops, slow and steady tanks, our heads kept low as we traverse the murky waters and head down into the mines. we value participation, removing barriers, decentralizing the control and letting each do he pleases. we're all about having a good time, making the most quality memes, writing good code, sharing and cooperating, and preserving transactions on a distributed private blockchain. we are turtlecoin. fun, safe, easy.

...and memes and pizza and turbocharged lambo moon cruisers and 'fuck' and beer and vices. dec 9th, 2017 was the start of a celebration and the party don't stop.

**Comment by rybofy** [comment]
That is probably the coolest thing ive ever read @BebopTC! And 100% agree

**Comment by bd-ongithub** [comment]
As a brand manifesto, I feel @BebopTC totally hit this out of the park. It's very self-reflecting of the actual personality of the Turtlecoin. Personally, I feel the design and graphic influences he plays upon in his description can be very fun to develop into finalized executions, at least on my end! 

Visually (and hopefully overall communication-wise as we develop this further) it can actually prove to be very disruptive - or at the very least pretty unique - when looking at the market as a whole. Asides from dogecoin which is basically comic sans + the iconic shiba inu, most if not all cryptocurrency projects with a large following really aim towards a very polished, sleek, futuristic, and/or minimalist image. A retro/grunge/technopunk approach seems to be surprisingly missing in the mix(ins), at least from my experience :)

One thing I personally see as more of a challenge/goal than an actual current fact is claiming to be the end-all, be-all meme coin, considering there's 'many wows, such history' to other project(s). Still, definitely something to shoot for.

I think I'm gonna work on setting up a visual mood board based on this manifesto, since it does paint quite the picture in my head, so getting a grouping of stuff that inspires these visuals should be helpful. To touch upon my past post, I still think touching upon the questions I laid out would still be good down the road.

**Comment by bd-ongithub** [comment]
And here's said moodboard.
![trtl---moodboard](https://user-images.githubusercontent.com/35929209/36347966-fc2fe63e-1433-11e8-845b-8927ef799220.jpg)

**Comment by RocksteadyTC** [comment]
@BebopTC and I were impressed with the mood board. I'd like to see how this idea takes shape, and let's see what we can do to merge some of this community spirit into the visual theme without alienating those looking for a confident, proper presentation.

**Comment by bd-ongithub** [comment]
Hey @RocksteadyTC and @BebopTC glad you guys liked it! 

I know you two are getting pulled in all sorts of different directions and perhaps scheduling a meeting right away might be a bit draining when there's other platform focused stuff that needs priority, so a good start would be if you both can take some time on and off to answer the branding questionnaire (even if it's just a question or two at a time, doesn't need to be filled out all at once).

I really believe this would be great for the marketing group in general, plus on my end it really allows me to focus just exactly how to run with this moodboard to concept something interesting without throwing out a really farfetched design/communication proposal that may end up being too extreme (already saw some comments tying the moodboard to the hacker known as 4chan, which had me chuckling for a bit), as well as having it follow existing brand guidelines well enough to do them justice.

**Comment by RocksteadyTC** [comment]
**What does TurtleCoin do?**
**You have 30 seconds to explain the project to potential investor/adopter, what would you say?**
_TurtleCoin is a network that aims to simplify the entire process of using, owning, and building on these types of cryptoasset networks. We hope to some day have private smart contracts, which are like network based programs you can interact with that don't require people to operate._
_"Top floor, please"_ 


**What unique problem do you solve?**
_We don't solve any unique technical problems that have fancy names, yet, but we aim to create the ability to run private autonomous code that does not require a person to administer, like a business or a game or store._


**Who are your 3 main competitors (direct or indirect)? What do you like about their presence? What do you dislike about their brand identity?**
_We could be considered to be in the same class as other ring signature based privacy projects, and hope to come day be held in the same light as different networks that run smart contracts, which we intend to implement in a more privacy oriented way. We like the degree of hard research going in to other concepts. We hope to provide an easier point of entry to the cryptoasset space for new users, and be more of a launchpad for creative ideas and innovative concepts._


**What is TurtleCoin's Unique Selling Proposition?**
_ If I were talking to someone in an elevator, like you mentioned, I'd simply state that if they like what they see when they check us out and maybe want to grow and learn with us, go ahead and say hello and stick around to kick the tires a bit; conversely if you don't have the time for something new and just want something with more track record, we appreciate that, and would happily recommend they try Bitcoin, or another network with more history._


**Is there a unique story behind your business or business name or logo?**
_Taken from the main web site's origin story,_
_One chilly December night, RockSteady and Bebop were having a beer too many while discussing the madness around fake, flimsy ICOs with no product. Bebop surmised that it just wouldn't be possible to put out a real project now days without a serious team, doing serious marketing, and bringing in serious funding.
RockSteady leaned in closely, stroked his beard intently, and had only one response..
"Hold my beer."
That night, December 9th, 2017, TurtleCoin was born.
Why?
Too many projects are bringing in too much funding despite most having no damn product. We figured, "why not do the whole thing in reverse? Start with a fun project, make it as simple and accessible as possible, and let it grow on its own."
We have a strong belief in our no-funding strategy; if we make a cool product, the value will create itself. If you think we're wrong, we're happy to talk about it._


**Share 5 adjective or words that best describe your company**
_Fast, Easy, Private, Nimble, Welcome!_


**Describe your ideal TurtleCoin adopter/user.**
_In its current state, we are attractive to someone who's a tinkerer, who either has something to learn or something to teach the community. We work every day for this to be a welcoming platform where anyone can hit the ground running and implement their ideas and see their work come to life  immediately, and we want to give them the skills to accomplish this._


**What is the primary message do you want to convey to an adopter/user?**
_We want to hear your ideas, and help you give them life. As a new project, there will be a few rough edges but we turn these into learning experiences, and we like working with each other to hammer them out together as ideas and then put them into action._ 


**Share 5 adjective or words that best describe your ideal adopter/user.**
_friendly, humble, fearless, eager, TURTLE!_

**Comment by RocksteadyTC** [comment]
Thanks @adj-gh

**Comment by bd-ongithub** [comment]
Heya @RocksteadyTC and @BebopTC 

Thanks for these answers to the questionnaire! Did you both work together on them or will Bebop be submitting his answers separately?

If possible, I'd like to dig a little deeper into these answers to see what more we can gleam from this exercise, check out my notes below:

**What does TurtleCoin do?**
You have 30 seconds to explain the project to potential investor/adopter, what would you say?
TurtleCoin is a network that aims to simplify the entire process of using, owning, and building on these types of cryptoasset networks. We hope to some day have private smart contracts, which are like network based programs you can interact with that don't require people to operate.
"Top floor, please"

_Generally, you want to convey one single talking point/idea in a 30 second window to make sure the point is as focused as possible. Right now I'm seeing two separate pitches: 1) A network designed in a way that using it, owning (running?) it, and building on it becomes simple or simpler than current standards. and 2) Creating a network that offers private smart contracts on its platform to enable a new level of privacy-focused automation
Can this be condensed into a single talking point? For example: TurtleCoin is a a privacy-based cryptoasset platform designed to be simple to use, operate, and develop upon._

**What unique problem do you solve?**
We don't solve any unique technical problems that have fancy names, yet, but we aim to create the ability to run private autonomous code that does not require a person to administer, like a business or a game or store.

_The less fancy sounding the better! So with that in mind, what base problem does private, self-administering and autonomous code seek to solve?
For example, in my view a base problem that cryptocurrencies in general seek to solve would be the transfer of value between two people, no matter where they are, without needing a trusted third party as an intermediary._

**Who are your 3 main competitors (direct or indirect)? What do you like about their presence? What do you dislike about their brand identity?**
We could be considered to be in the same class as other ring signature based privacy projects, and hope to come day be held in the same light as different networks that run smart contracts, which we intend to implement in a more privacy oriented way. We like the degree of hard research going in to other concepts. We hope to provide an easier point of entry to the cryptoasset space for new users, and be more of a launchpad for creative ideas and innovative concepts.

_Would be good to name some names here to focus further. Note that competitors doesn't mean we're actively gunning for anyone, simply other (perhaps more well known) projects that exist in our realm/sphere that you think are doing things right (and also in the same way seeing things in those projects that you'd do differently).
For example, could Ethereum or Stellar be considered competing smart contract platforms? What about Monero or Bytecoin when it comes to privacy?_

**What is TurtleCoin's Unique Selling Proposition?**
_ If I were talking to someone in an elevator, like you mentioned, I'd simply state that if they like what they see when they check us out and maybe want to grow and learn with us, go ahead and say hello and stick around to kick the tires a bit; conversely if you don't have the time for something new and just want something with more track record, we appreciate that, and would happily recommend they try Bitcoin, or another network with more history._

_Well, this isn't exactly an attempt to revise the 30 second elevator pitch. Think of a unique selling proposition as something the project/brand has that others don't.
Basically, what would you like TurtleCoin to be known for? What does it stand for?_

**Is there a unique story behind your business or business name or logo?**
Taken from the main web site's origin story,
One chilly December night, RockSteady and Bebop were having a beer too many while discussing the madness around fake, flimsy ICOs with no product. Bebop surmised that it just wouldn't be possible to put out a real project now days without a serious team, doing serious marketing, and bringing in serious funding.
RockSteady leaned in closely, stroked his beard intently, and had only one response..
"Hold my beer."
That night, December 9th, 2017, TurtleCoin was born.
Why?
Too many projects are bringing in too much funding despite most having no damn product. We figured, "why not do the whole thing in reverse? Start with a fun project, make it as simple and accessible as possible, and let it grow on its own."
We have a strong belief in our no-funding strategy; if we make a cool product, the value will create itself. If you think we're wrong, we're happy to talk about it.

_So this is the already published version, but is there anything more to the story we can play with? Check out the Johnnie Walker ad I shared, the devil is in the details (if you'd care to provide any more of course, if not we can just run with this)_

**Share 5 adjective or words that best describe your company**
Fast, Easy, Private, Nimble, Welcome!

_Can you add 2 more? Let's omit fast, easy, and safe for now._

**Describe your ideal TurtleCoin adopter/user.**
In its current state, we are attractive to someone who's a tinkerer, who either has something to learn or something to teach the community. We work every day for this to be a welcoming platform where anyone can hit the ground running and implement their ideas and see their work come to life immediately, and we want to give them the skills to accomplish this.

_Let's go back to describing the person a little more. Right now we know this ideal adopter likes to tinker with things, what else? Are they programming-savvy, and if so how much? Are they introverts/extroverts? Where do they hang out? What sort of things do they like, and what kind of stuff do they consider important in their lives?_

**What is the primary message that you want to convey to an adopter/user?**
We want to hear your ideas, and help you give them life. As a new project, there will be a few rough edges but we turn these into learning experiences, and we like working with each other to hammer them out together as ideas and then put them into action.

_How can this be condensed further? Think a year, maybe 3 or 4 in the future, if someone were to mention TurtleCoin to someone, what's the first thought that'd pop into their head? How would that make them feel?_

**Share 5 adjective or words that best describe your ideal adopter/user.**
friendly, humble, fearless, eager, TURTLE!

_In your view, what does being a turtle represent?_

Again, thanks for taking some time with this branding exercise, the more insights the better :)

**Comment by holytastyguacamole** [comment]
Hey peeps, @RocksteadyTC | @BebopTC | @bd-ongithub 

I personally think you may be overthinking this and not really putting your best foot forward. 

From my point of view as a non-technical person, the most important aspect of TurtleCoin which appealed to me straight away is the 2 decimal places. The answers to the questions below actually flow naturally if you take this into account: 

**What does TurtleCoin do?**
TurtleCoin aims to bring cryptocurrencies to the everyday person. We have made our coin simple to understand and only have two decimal places. Why make things more complicated than they need to be? Easy to understand and easy to use! 

**What unique problem do you solve?**
While we do not solve any unique technical problem, we are all about making things easier to use and understand. Instead of asking people to pay for a cup of coffee with 0.000000123 coins, our vision is for them to use 10 shells, or 5.2 shells. Simple. Turtlecoin comes to meet people where they are at.

**Who are your 3 main competitors (direct or indirect)? What do you like about their presence? What do you dislike about their brand identity?**
Monero - As described above, too many decimal places
Aeon - (dont know much about aeon)
Dogecoin - too much of a joke-coin

**What is TurtleCoin's Unique Selling Proposition?**
Ease of use. Period. 
Now this may not be true just yet, but doesnt mean that cannot be the vision moving forward

**Is there a unique story behind your business or business name or logo?**
We love turtles!

**Share 5 adjective or words that best describe your company**
Your words seem fine. I disagree, with leaving easy out though, as per the point above. Perphaps 'Simple'

**Describe your ideal TurtleCoin adopter/user.**
Eventually we would like to see everyday people use our coin. Your mum, dad and grandparents, crypto shouldn't be restricted to people who are good with numbers or coding!

**What is the primary message that you want to convey to an adopter/user?**
Our main goal is to make Turtlecoin accessible to everyone. We would love to have you join us now to help us make it your crypto of choice.

**Share 5 adjective or words that best describe your ideal adopter/user.**
Friendly, humble, fearless, eager, TURTLE!

I think the concept of two decimal places is very much undersold in all of conversations. Possibly because we have grown used to dealing in so many decimals, but like Rocksteady said 'you want me to do that kind of arithmetic when I buy a cup of coffee? gtfo' 

Anyway, hope that helps.

**Comment by RocksteadyTC** [comment]
![image](https://user-images.githubusercontent.com/34389545/36773475-6f3fb4ea-1c1f-11e8-8781-c132b4e0038c.png)
This is not okay, and this is not how I chose to answer the question for a reason. I go through great lengths not to throw dirt on anybody else's projects or make any comparisons on a non-technical basis. This question is no acception.

**Comment by RocksteadyTC** [comment]
@bd-ongithub I answered these on my own

**Comment by lamigo22** [comment]
we should start a donation campaign and workt with them together

https://twitter.com/CICTMAR/status/971058294147158018

Its a Civil Association that works in saving of marine turtles in Venezuela
maybe we can get some traction and good marketing within the media?
